### PR TITLE
Closes #332: Move files tab to separate backing bean

### LINKED ISSUE #332: Extract file table tab in DatasetPage related code into separate backing bean

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: dataverse-webapp/src/main/webapp/filesFragment.xhtml [hunk 1]
```diff
@@ -245,84 +244,84 @@
...
                     <!-- guest book or terms of use, etc. enabled - open "download popup" first: -->
-                    <p:commandLink rendered="#{DatasetPage.downloadPopupRequired}"
+                    <p:commandLink rendered="#{datasetFilesTab.downloadPopupRequired}"
                                     type="button" styleClass="btn btn-default btn-download"
-                                    disabled="#{false and DatasetPage.lockedFromDownload}"
-                                    action="#{DatasetPage.validateFilesForDownload(true, false)}"
-                                    update="@form" >
+                                    disabled="#{false and datasetFilesTab.lockedFromDownload}"
+                                    action="#{datasetFilesTab.validateFilesForDownload(true, false)}"
+                                    update="@form" oncomplete="">
```
Comment (MrK191) [inline dataverse-webapp/src/main/webapp/filesFragment.xhtml]: Is blank oncomplete needed?

File: dataverse-webapp/src/main/webapp/filesFragment.xhtml [hunk 2]
```diff
@@ -245,84 +244,84 @@
...
                                 </p:commandLink>
                                 <!-- guest book or terms of use, etc. enabled - open "download popup" first: -->
-                                <p:commandLink rendered="#{DatasetPage.downloadPopupRequired}"
-                                                disabled="#{false and DatasetPage.lockedFromDownload}"
-                                                action="#{DatasetPage.validateFilesForDownload(true, true)}"
-                                                update="@form" >
+                                <p:commandLink rendered="#{datasetFilesTab.downloadPopupRequired}"
+                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
+                                                action="#{datasetFilesTab.validateFilesForDownload(true, true)}"
+                                                update="@form" oncomplete="">
```
Comment (MrK191) [inline dataverse-webapp/src/main/webapp/filesFragment.xhtml]: Is blank oncomplete needed?

File: dataverse-webapp/src/main/webapp/filesFragment.xhtml [hunk 3]
```diff
@@ -245,84 +244,84 @@
...
                                 </p:commandLink>
                                 <!-- guest book or terms of use, etc. enabled - open "download popup" first: -->
-                                <p:commandLink rendered="#{DatasetPage.downloadPopupRequired}"
-                                                disabled="#{false and DatasetPage.lockedFromDownload}"
-                                                action="#{DatasetPage.validateFilesForDownload(true, false)}"
-                                                update="@form" >
+                                <p:commandLink rendered="#{datasetFilesTab.downloadPopupRequired}"
+                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
+                                                action="#{datasetFilesTab.validateFilesForDownload(true, false)}"
+                                                update="@form" oncomplete="">
```
Comment (MrK191) [inline dataverse-webapp/src/main/webapp/filesFragment.xhtml]: Is blank oncomplete needed?

**Review by MrK191 — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/webapp/filesFragment.xhtml (hunk 1 above)
Comment (madryk) [reply dataverse-webapp/src/main/webapp/filesFragment.xhtml]: no

File: dataverse-webapp/src/main/webapp/filesFragment.xhtml (hunk 2 above)
Comment (madryk) [reply dataverse-webapp/src/main/webapp/filesFragment.xhtml]: no

File: dataverse-webapp/src/main/webapp/filesFragment.xhtml (hunk 3 above)
Comment (madryk) [reply dataverse-webapp/src/main/webapp/filesFragment.xhtml]: no, it's not needed

**Review by MrK191 — APPROVED** [review]

**Pull request merged**